### ISSUE TITLE
stripQuerystring configuration results in an error

### ISSUE THREAD

**Issue by geramy92** [cgiffard/node-simplecrawler#332]
## What happened?
I tried to use 
```js
let crawler = new Crawler(...);
 crawler.stripQuerystring = true;
 ...
```
but it results in an error.

## What should have happened?
It should worked like documented.

## Steps to reproduce the problem
see What happened. After that you get the error I added below

##Error
```
TypeError: URI.removeQuery() accepts an object, string, RegExp as the first parameter
    at Function.URI.removeQuery (C:\Users\some_user\Documents\Webstrom\crawler\node_modules\urijs\src\URI.js:795:13)
    at URI.p.removeQuery (C:\Users\some_user\Documents\Webstrom\crawler\node_modules\urijs\src\URI.js:1789:9)
    at Crawler.processURL (C:\Users\some_user\Documents\Webstrom\crawler\node_modules\simplecrawler\lib\crawler.js:743:24)
    at Crawler.start (C:\Users\some_user\Documents\Webstrom\crawler\node_modules\simplecrawler\lib\crawler.js:436:29)
    at Object.<anonymous> (C:\Users\some_user\Documents\Webstrom\crawler\cli.js:131:9)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
```

**Comment by ttessarolo** [comment]
I have the same problem...

**Comment by fredrikekelund** [comment]
Thanks for reporting this and sorry for the slow response! This was an oversight on my part - the logic for this option was changed before the 1.0.0 release and we didn't have any unit tests for it. I've fixed it now and released 1.0.2 to npm!
